answers/Java/@thecarisma/N02Lottery.java

```diff
@@ -0,0 +1,40 @@
+
+import java.io.IOException;
+import java.util.Random;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/*
+ * The MIT License
+ *
+ * Copyright 2019 Azeez Adewale <azeezadewale98@gmail.com>.
+ *
+ */
+
+/**
+ *
+ * @author Azeez Adewale <azeezadewale98@gmail.com>
+ */
+public class N02Lottery {
+    
+    public static void main(String[] args){
+        try {
```

Comment:
The program is supposed to repeat itself, so perhaps, have a while loop around the try {...} ?